### PR TITLE
DEP-256: Support v2.8.0 of the iOS SDK

### PR DESCRIPTION
**Pull request by belkhadirAnas**
### Purpose

Support v2.8.0 of the iOS SDK
Update RN to v1.9.0

### External References (e.g. Jira / Zeplin / Confluence)
[DEP-256](https://lampkicking.atlassian.net/browse/DEP-256)

### CODE REVIEW

**Review by rgcriado — APPROVED** [review]

File: example/ios/example/Info.plist [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>1.9.0</string>
```
Comment (mariasenosiain) [inline example/ios/example/Info.plist:20]: Why this change? `$(MARKETING_VERSION)` was introduced in this PR https://github.com/getyoti/yoti-doc-scan-react-native/pull/15/files and I thought it was useful to get the version number in a more dynamic way here 🤔

File: example/ios/example.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mariasenosiain) [inline example/ios/example.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: Just a doubt! Why .xcworlspacedata and this plist files are included in this PR? Previously they were not needed 👀

File: ios/rnyotidocscan.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```
Comment (mariasenosiain) [inline ios/rnyotidocscan.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:4]: Same than the previous question 👀

Comment (belkhadirAnas) [reply ios/rnyotidocscan.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:4]: It's created by Xcode automatically you can check. on [stackOverflow](https://stackoverflow.com/questions/50367896/ideworkspacechecks-plist-file-suddenly-appear-after-updated-xcode)

File: example/ios/example.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 2 above)
Comment (belkhadirAnas) [reply example/ios/example.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: It's created by Xcode automatically you can check. on [stackOverflow](https://stackoverflow.com/questions/50367896/ideworkspacechecks-plist-file-suddenly-appear-after-updated-xcode)

**Comment by Zzhelyaz** [comment]
Tested the Document capture flow with NFC, Supplementary document flow and Liveness. All good!

**Review by Zzhelyaz — APPROVED** [review]

File: example/ios/example/Info.plist (hunk 1 above)
Comment (belkhadirAnas) [reply example/ios/example/Info.plist:20]: As I remember Anders told me always to update MARKETING VERSION this why I did in a static way

**Review by mariasenosiain — APPROVED** [review]
